pixart.js

```diff
@@ -1 +1,30 @@
 
+$(document).ready(function canvas() {
+    for (i = 0; i<= 5000; i++)
```

Comment:
Make sure to properly indent and space your code. Readibility is super important.